src/components/league/current-round/index.tsx

```diff
@@ -33,15 +34,26 @@ export const CurrentRoundView = ({
                 </TouchableOpacity>
             </View>
             <ScrollView>
-                {Object.values(currentGame.players)
-                .map((player: any, index: number) => (
+                <Text
+                    style={{
+                        textAlign: 'center',
```

Comment:
we'll spend some fixing tech debt at some point :@. now is the time to move fast (and accumulate tech debt :@)